hq/app/logic/IamUnrecognisedUsers.scala

```diff
@@ -112,39 +112,23 @@ object IamUnrecognisedUsers extends Logging {
     ))
   }
 
-  private def removeAccountPasswords(
-    account: AwsAccount,
-    vulnerableUsers: List[HumanUser],
+  def removeAccountPasswords(
+    accountUnrecognisedUsers: AccountUnrecognisedUsers,
     iamClients: AwsClients[AmazonIdentityManagementAsync]
   )(implicit ec: ExecutionContext): Attempt[List[Option[DeleteLoginProfileResult]]] = {
-    val results = Attempt.traverse(vulnerableUsers)(user => deleteLoginProfile(account, user.username, iamClients))
+    val results = Attempt.traverse(accountUnrecognisedUsers.unrecognisedUsers)(user => deleteLoginProfile(accountUnrecognisedUsers.account, user.username, iamClients))
     results.tap {
       case Left(failure) =>
         logger.error(s"failed to delete at least one password: ${failure.logMessage}")
         Cloudwatch.putIamRemovePasswordMetric(ReaperExecutionStatus.failure, 1)
       case Right(success) =>
-        logger.info(s"passwords deleted for ${vulnerableUsers.map(_.username).mkString(",")}")
+        logger.info(s"passwords deleted for ${accountUnrecognisedUsers.unrecognisedUsers.map(_.username).mkString(",")}")
         Cloudwatch.putIamRemovePasswordMetric(ReaperExecutionStatus.success, success.flatten.length)
     }
   }
 
-  def disableAccountUsers(
```

Comment:
this method is no longer needed because we do this work at the edge of the job in a for comprehension